pybamm/models/full_battery_models/base_battery_model.py

```diff
@@ -263,7 +263,7 @@ def __init__(self, extra_options):
                 "stress and reaction-driven",
             ],
             "number of MSMR reactions": ["none"],
-            "open-circuit potential": ["single", "current sigmoid", "MSMR"],
+            "open-circuit potential": ["single", "current sigmoid", "MSMR", "Plett"],
```

Comment:
Is it okay to call it "Plett" OCP when it's from the Wycisk modified plett paper?

Reply:
Can we give it a more informative name like "Differential capacity varying" (or something more concise!). Then we can point to the paper in the docs.

Reply:
How about "Differential capacity hysteresis state" as "DCHS" ?